code/game/objects/items/needle.dm

```diff
@@ -100,6 +100,8 @@
 		if((armor_value == 0 && skill_level > 0) || (armor_value > 0 && skill_level > 1)) //If not armor but skill level at least 1 or Armor and skill level at least 2
 			user.visible_message(span_info("[user] repairs [I]!"))
 			I.repair_damage(skill_multiplied)
+			if(prob(33))
```

Comment:
why is this a flat prob? could use sewing skill as a modifier. Is the needle with medicine the same?

Reply:
Done